openapi-security/src/main/java/com/networknt/openapi/UnifiedSecurityHandler.java

```diff
@@ -106,43 +116,107 @@ public void handleRequest(HttpServerExchange exchange) throws Exception {
+                                // and then call the right handler if the type is configured as true.
                                 Map<String, HttpHandler> handlers = Handler.getHandlers();
-                                if(pathPrefixAuth.isJwt() && pathPrefixAuth.isSwt()) {
-                                    // both jwt and swt are enabled.
-                                    boolean isJwt = StringUtils.isJwtToken(authorization);
-                                    if(logger.isTraceEnabled()) logger.trace("Both jwt and swt are true and check token is jwt = {}", isJwt);
-                                    if(isJwt) {
-                                        JwtVerifyHandler handler = (JwtVerifyHandler) handlers.get(JWT);
-                                        if (handler == null) {
-                                            logger.error("Cannot find JwtVerifyHandler with alias name jwt.");
-                                            setExchangeStatus(exchange, HANDLER_NOT_FOUND, "com.networknt.openapi.JwtVerifyHandler@jwt");
-                                            exchange.endExchange();
-                                            return;
-                                        } else {
-                                            // if the handler is not enabled in the configuration, break here to call next handler.
-                                            if(!handler.isEnabled()) {
-                                                break;
-                                            }
-                                            // get the jwkServiceIds list.
-                                            if (handler.handleJwt(exchange, pathPrefixAuth.getPrefix(), reqPath, pathPrefixAuth.getJwkServiceIds())) {
-                                                // verification is passed, go to the next handler in the chain.
-                                                break;
+                                // remove the Bearer prefix to get the token.
+                                String token = authorization.substring(7);
+                                // first to identify the token type.
+                                boolean isJwtOrSjwt = StringUtils.isJwtToken(token);
+                                if(logger.isTraceEnabled()) logger.trace("Bearer token is jwt or sjwt = {}", isJwtOrSjwt);
+                                if(isJwtOrSjwt) {
+                                    // if sjwt is true, need to parse the token to see if it has scope or scp claim to identify if it is a jwt or sjwt.
+                                    if(pathPrefixAuth.isSjwt()) {
+                                        if(logger.isTraceEnabled()) logger.trace("SJWT is enabled.");
+                                        // check if jwt is true.
+                                        if(pathPrefixAuth.isJwt()) {
+                                            if(logger.isTraceEnabled()) logger.trace("JWT is enabled along with swt.");
```

Comment:
Trace msg should be `JWT is enabled along with SJWT.`

Reply:
Thanks for the catch. It is updated.